pkg/doc/did/doc.go

```diff
@@ -118,6 +120,31 @@ type PublicKey struct {
 	jsonWebKey *jose.JWK
 }
 
+// NewPublicKeyFromBytes creates a new PublicKey based on raw public key bytes.
+func NewPublicKeyFromBytes(id, kType, controller string, value []byte) *PublicKey {
+	return &PublicKey{
+		ID:         id,
+		Type:       kType,
+		Controller: controller,
+		Value:      value,
+	}
+}
+
+// NewPublicKeyFromJWK creates a new PublicKey based on JSON Web Key.
+func NewPublicKeyFromJWK(id, kType, controller string, jwk *jose.JWK) *PublicKey {
+	return &PublicKey{
+		ID:         id,
+		Type:       kType,
+		Controller: controller,
+		jsonWebKey: jwk,
+	}
+}
```

Comment:
nit: These functions do nothing. I would suggest to make `jsonWebKey` field public and remove these functions.

Reply:
Oh man, this is how it looked before, but this can confuse, please see starting comment https://github.com/hyperledger/aries-framework-go/pull/1597#discussion_r407567036 and scroll down :)

Reply:
I think that the specific problem is that we Value/Type has overlap with JWK contents. But I am confused about https://github.com/hyperledger/aries-framework-go/pull/1597#r408393621